rdrf/registry/patients/admin_forms.py

```diff
@@ -154,7 +154,19 @@ def save(self, commit=True):
 class PatientStageForm(forms.ModelForm):
     class Meta:
         model = PatientStage
-        fields = "__all__"
+        fields = ["registry", "name", "allowed_prev_stages", "allowed_next_stages"]
+
+    def __init__(self, *args, **kwargs):
+        super(PatientStageForm, self).__init__(*args, **kwargs)
+        if self.instance.pk:
+            self.fields['registry'].disabled = True
+        registry = self.instance.registry or None
+        stage_qs = (
```

Comment:
`registry__isnull=True` is the same as `registry=None`, so you can just use `registry=registry` ie. you don't need an `if` at all.

Reply:
Updated.